Earthfile

```diff
@@ -0,0 +1,112 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Without syntax highlighting it's pretty hard to read, is there perhaps an editor plugin for earthfiles?

Reply:
Yes - see https://earthly.dev/get-earthly